lumicks/pylake/fitting/fit.py

```diff
@@ -243,8 +256,8 @@ def fit(self, show_fit=False, **kwargs):
         if np.any(out_of_bounds):
             raise ValueError(
                 f"Initial parameters {self.params.keys()[fitted][out_of_bounds]} are outside the "
-                f"parameter bounds. Please set value, lower_bound and upper_bound for these parameters"
-                f"to consistent values."
+                "parameter bounds. Please set value, `lower_bound` and `upper_bound` for these "
+                "parameters to consistent values."
```

Comment:
👍 

I have omitted the backticks, since I think they belong in the docstring, but not in the exception itself. I also don't think the comma is needed?